packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -332,7 +332,7 @@ const ZoomVideoApiAdapter = (credential: CredentialPayload): VideoApiAdapter =>
           },
           body: JSON.stringify(await translateEvent(event)),
         });
-        
+
```

Comment:
This is OK for now, but unrelated files do clutter, will just mark as "viewed" 👍